src/stitch-tiles.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const { DEFAULT_LENGTH } = require('../utils/constants');
+const mapnik = require('mapnik');
+const { PNG } = require('pngjs');
+
+async function stitchTiles(tileArray, length = DEFAULT_LENGTH, decode = true) {
+  const tiles = await tileArray;
+
+  return new Promise((resolve, reject) => {
+    mapnik.blend(tiles,
+      {
+        format: 'png',
+        width: length,
+        height: length,
+        reencode: true
+      }, (err, img) => {
+        if (err) return reject(err);
+        return resolve(
+          {
+            decoded: decode ? PNG.sync.read(img) : null,
+            original: img.toString('base64')
```

Comment:
Thoughts on just returning the base64 encoded string? Personally, I feel like consumers should create the png if they need to. quilt can keep pretty lean by not including `pngjs` as a dependency.

Reply:
Ooh, I like where your head's at. I _would_ like to keep quilt as light as possible, so I can definitely get on board with pushing responsibility for decoding to downstream consumers. 👍

Reply:
Switching to returning just the raw buffer in https://github.com/mapbox/quilt/pull/1/commits/9abde5f45017c0ac14c494e351ad572794241092